src/Nethermind/Nethermind.Evm/EvmStack.cs

```diff
@@ -208,6 +258,17 @@ public Address PopAddress()
             return new Address(_bytes.Slice(Head * 32 + 12, 20).ToArray());
         }
 
+        private ref byte PopBytesByRef()
```

Comment:
maybe better call thes GetStackRef and only move head in the PopUInt

Reply:
Dunno. May rename if you wish. I treat `EvmStack` as a whole and this is `private`. 
Let me know @tkstanczak